<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-07T09:35:46 IST" name="Suite" finished-at="2024-06-07T09:36:25 IST" duration-ms="39412">
    <groups>
    </groups>
    <test started-at="2024-06-07T09:35:46 IST" name="SubmitOrder" finished-at="2024-06-07T09:36:14 IST" duration-ms="28095">
      <class name="Frameworks.SeleniumFrameWorks.StandAlone2">
        <test-method is-config="true" signature="launchingtheapplication()[pri:0, instance:Frameworks.SeleniumFrameWorks.StandAlone2@7f8a9499]" started-at="2024-06-07T09:35:46 IST" name="launchingtheapplication" finished-at="2024-06-07T09:35:55 IST" duration-ms="9124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchingtheapplication -->
        <test-method signature="submitorder(java.util.HashMap)[pri:0, instance:Frameworks.SeleniumFrameWorks.StandAlone2@7f8a9499]" started-at="2024-06-07T09:35:55 IST" name="submitorder" data-provider="getData" finished-at="2024-06-07T09:35:59 IST" duration-ms="4436" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Hari@799, email=hareesh123@gmail.com, productName=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitorder -->
        <test-method is-config="true" signature="closingbrowser()[pri:0, instance:Frameworks.SeleniumFrameWorks.StandAlone2@7f8a9499]" started-at="2024-06-07T09:35:59 IST" name="closingbrowser" finished-at="2024-06-07T09:36:01 IST" duration-ms="1793" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingbrowser -->
        <test-method is-config="true" signature="launchingtheapplication()[pri:0, instance:Frameworks.SeleniumFrameWorks.StandAlone2@7f8a9499]" started-at="2024-06-07T09:36:01 IST" name="launchingtheapplication" finished-at="2024-06-07T09:36:09 IST" duration-ms="8106" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchingtheapplication -->
        <test-method signature="submitorder(java.util.HashMap)[pri:0, instance:Frameworks.SeleniumFrameWorks.StandAlone2@7f8a9499]" started-at="2024-06-07T09:36:09 IST" name="submitorder" data-provider="getData" finished-at="2024-06-07T09:36:13 IST" duration-ms="4104" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Hari@799, email=harish234@gmail.com, productName=ADIDAS ORIGINAl}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitorder -->
        <test-method is-config="true" signature="closingbrowser()[pri:0, instance:Frameworks.SeleniumFrameWorks.StandAlone2@7f8a9499]" started-at="2024-06-07T09:36:13 IST" name="closingbrowser" finished-at="2024-06-07T09:36:14 IST" duration-ms="298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingbrowser -->
      </class> <!-- Frameworks.SeleniumFrameWorks.StandAlone2 -->
    </test> <!-- SubmitOrder -->
    <test started-at="2024-06-07T09:36:14 IST" name="Error Validations" finished-at="2024-06-07T09:36:25 IST" duration-ms="10972">
      <class name="Frameworks.SeleniumFrameWorks.ErrorValidations">
        <test-method is-config="true" signature="launchingtheapplication()[pri:0, instance:Frameworks.SeleniumFrameWorks.ErrorValidations@6d026701]" started-at="2024-06-07T09:36:14 IST" name="launchingtheapplication" finished-at="2024-06-07T09:36:17 IST" duration-ms="3455" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchingtheapplication -->
        <test-method signature="LoginValidation()[pri:0, instance:Frameworks.SeleniumFrameWorks.ErrorValidations@6d026701]" started-at="2024-06-07T09:36:17 IST" name="LoginValidation" finished-at="2024-06-07T09:36:18 IST" duration-ms="517" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginValidation -->
        <test-method is-config="true" signature="closingbrowser()[pri:0, instance:Frameworks.SeleniumFrameWorks.ErrorValidations@6d026701]" started-at="2024-06-07T09:36:18 IST" name="closingbrowser" finished-at="2024-06-07T09:36:18 IST" duration-ms="147" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingbrowser -->
        <test-method is-config="true" signature="launchingtheapplication()[pri:0, instance:Frameworks.SeleniumFrameWorks.ErrorValidations@6d026701]" started-at="2024-06-07T09:36:18 IST" name="launchingtheapplication" finished-at="2024-06-07T09:36:21 IST" duration-ms="3227" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchingtheapplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:Frameworks.SeleniumFrameWorks.ErrorValidations@6d026701]" started-at="2024-06-07T09:36:21 IST" name="ProductErrorValidation" finished-at="2024-06-07T09:36:25 IST" depends-on-methods="LoginValidation()" duration-ms="3181" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at Frameworks.SeleniumFrameWorks.ErrorValidations.ProductErrorValidation(ErrorValidations.java:42)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1623)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="closingbrowser()[pri:0, instance:Frameworks.SeleniumFrameWorks.ErrorValidations@6d026701]" started-at="2024-06-07T09:36:25 IST" name="closingbrowser" finished-at="2024-06-07T09:36:25 IST" duration-ms="270" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingbrowser -->
      </class> <!-- Frameworks.SeleniumFrameWorks.ErrorValidations -->
    </test> <!-- Error Validations -->
  </suite> <!-- Suite -->
</testng-results>
